__tests__/commands/add.js

```diff
@@ -114,6 +114,16 @@ test.concurrent('install from github', async () => {
   await runAdd(['substack/node-mkdirp#master'], {}, 'install-github');
 });
 
+test.concurrent('install from github with invalid version should fail', async () => {
+  let message = '';
+  try {
+    await runAdd(['yarnpkg/example-yarn-package#invalid-package-json-version'], {}, 'install-github');
+  } catch (err) {
+    message = err.message;
+  }
+  expect(message).toEqual(expect.stringContaining('invalid package version'));
```

Comment:
Would be preferable to use `.rejects` and assert on the error message using `toThrow` or the like, but it looks like this is still problematic: https://github.com/facebook/jest/issues/3601